src/OpenConext/ProfileBundle/Resources/views/Introduction/overview.html.twig

```diff
@@ -0,0 +1,16 @@
+{% extends 'OpenConextProfileBundle::layout.html.twig' %}
+
+{% block content %}
+    <h1>{{ 'introduction.long_title'|trans }}</h1>
```

Comment:
Translation keys are global identifiers, let's put some form of namespace before each key (`profile.`).